### PR TITLE
Update tc base

### PR DESCRIPTION
**Pull request by jonasfj**
@jhford this refactors taskcluster-queue to use new features from taskcluster-base `0.8.1`.

 * All schemas to YAML
 * Remove ids from schemas
 * Removed old unused schemas
 * Use params to validate URL parameters
 * use schemaPrefix in API and exchanges rather than a constant
 * All scopes we check for is on Disjunctive Form

Please, do a quick review. All test pass (ignore travis-ci)...
@jhford, then please give the provisioner the same treatment and upgrade taskcluster-base... and adopt things like no ids, `schemaPrefix`, URL parameter validation (use same regex as me), etc.. 

In the future we should probably make more of this mandatory. But for now it fixes a lot of small bugs :)

### CODE REVIEW

File: queue/exchanges.js [hunk 1]
```diff
@@ -176,7 +174,7 @@ exchanges.declare({
     "using `createTask`."
   ].join('\n'),
   routingKey:         buildCommonRoutingKey(),
-  schema:             SCHEMA_PREFIX_CONST + 'task-defined-message.json#',
+  schema:             'task-defined-message.json#',
```
Comment (jhford) [inline queue/exchanges.js]: so much nicer to read!

**Comment by jhford** [comment]
Yep!  happy to be able to use this for the provisoiner.  Looks good to me.  Not sure if I should land or not, so I'll was until @jonasfj is online to let me know for sure.

**Comment by jonasfj** [comment]
merged. I assume I got an `r+` :)

**Comment by jhford** [comment]
you did!  :P

**Pull request merged**